.gitignore

```diff
@@ -23,6 +23,7 @@ Servo.app
 .config.mk.last
 /glfw
 capture_webrender/
+/components/compositing/webrender_revision.rs
```

Comment:
Is there a particular reason for routing the revision through rust code? could just generate a "wr.txt" and just copy it, alternatively

Reply:
Yes, I mentioned in PR body if this is an acceptable approach. I wasn't entirely sure about copying text file for cases if user downloaded nightly build wanted to capture it, does lookup generated text file would works with same path resolution to dev build? Thought having constant in build itself will makes those always working.

Reply:
Sorry, I don't understand the concern "does lookup generated text file would works with same path resolution to dev build". Please rephrase?

Reply:
I guess it's my misunderstanding mostly. in short, does packaged nightly build can lookup where wr.txt same as mach run on dev build?

Reply:
oh, I see. So it's a question to package builders on how to make sure "wr.txt" is included. @jdm do you know who forward this to?